blobfuse-nightly.yml

```diff
@@ -556,6 +556,17 @@ jobs:
           continueOnError: true
 
         - script:  |
+            sudo fusermount -u $(MOUNT_DIR)
+            ./build/blobfuse $(MOUNT_DIR) --tmp-path=$(TEMP_DIR) --config-file=$(BLOBFUSE_CFG_ADLS) --use-adls=true --file-cache-timeout-in-seconds=0 --streaming=true --max-block-per-file=3 --stream-cache-mb=500 -o ro
```

Comment:
humm good suggestion. let me see if we can convert this also to take total number of blocks instead of memory.  Initially I wrote code to have first block of different size and rest of the blocks of uniform size so counting on number of blocks was not good idea hence took the memory usage. But later that logic had some complexities so unified the block size.

Reply:
on second thought I think for end-user its easy to mention what cap they want to put on total memory instead of counting on number of blocks to be used and then checking x-size block and y-count and doing the airthmatic. We can discuss on what will be easier for customer to configure.

Reply:
wouldn't they already have to have some sort of calculation on their blocks given that they need to specify `blocks-per-file`? So they would have given this thought

Reply:
To make decision on memory usage, giving value of how much max memory application shall use is easier to think of. So giving total number of blocks needs more thought than memory usage. It just makes easy to user to decide. For blocks-per-file argument, yes they have done some airthmatic but it will be more on how much data they want to cache per file rather then how much memory we will consume. This will totally depend on their application as to what amount of data is sufficient for them to make any application level decisions.

Reply:
sounds good lets use cache-size-mb i have added it in azfuse as well